### DISCUSSION TITLE
Payment Methods Bank Account Number

### DISCUSSION THREAD

**Discussion by vajih** [mmohades/Venmo#51] [Q&A] [answered]
I am able to get the name and ID of the bank account however a few of my accounts share the same name but different bank account numbers. Is there any way to extract the bank account numbers using a Payment Methods function?

**Comment by mmohades** [comment, marked as answer]
Get the last four digits of your accounts. Here is an example
```python3
payment_methods = client.payment.get_payment_methods()
for payment_method in payment_methods:
    print(payment_method.to_json()['last_four'])
```
You may print the whole JSON to see more information.
